src/main/resources/hudson/plugins/build_timeout/operations/AbortAndRestartOperation/config_ja.properties

```diff
@@ -1,6 +1,6 @@
 # The MIT License
 #
-# Copyright (c) 2015, Fernando Migu�lez <fernando.miguelez@gmail.com>
+# Copyright (c) 2004-2016, Sun Microsystems, Inc., Seiji Sogabe
```

Comment:
Some how it is diaplayed as renamed from LikelyStuckTimeOutStrategy/config_es.properties even though it is actually commited as renamed from LikelyStuckTimeOutStrategy/config_ja.properties.
Anyway, LikelyStuckTimeOutStrategy/config_*.properties is unnecessary as ditto.